tests/invalid_models_tests/test_ordinary_fields.py

```diff
@@ -1057,3 +1058,61 @@ class Meta:
 
         errors = Model._meta.get_field("field").check(databases=self.databases)
         self.assertEqual(errors, [])
+
+
+@isolate_apps("invalid_models_tests")
+class ComputedFieldTests(TestCase):
+    @skipIfDBFeature("supports_computed_field")
+    def test_field_not_supported(self):
+        class Model(models.Model):
+            name = models.IntegerField()
+            field = models.ComputedField(expression=F("name"))
+
+        self.assertEqual(
+            Model._meta.get_field("field").check(databases={"default"}),
+            [
+                Error(
+                    "%s does not support ComputedFields." % connection.display_name,
+                    obj=Model._meta.get_field("field"),
+                    id="fields.E220",
+                ),
+            ],
+        )
+
+    @skipUnlessDBFeature("supports_computed_field")
+    @skipIfDBFeature("supports_computed_field_params")
+    def test_params_not_supported(self):
+        class Model(models.Model):
+            field = models.ComputedField(Value("value", output_field=TextField()))
```

Comment:
You shouldn't need `output_field` here, as `Value` can determine it automatically:

```suggestion
            field = models.ComputedField(Value("value"))
```